src/ONE.Models.CSharp/ONE.Models.CSharp.csproj

```diff
@@ -2,9 +2,9 @@
 
   <PropertyGroup>
     <!--For Release packages update the VersionPrefix and remove any VersionSuffix-->
-    <VersionPrefix>7.13.0</VersionPrefix>
+    <VersionPrefix>7.14.0</VersionPrefix>
     <!--For Feature packages update the VersionSuffix with the jira ticket number-->
-    <VersionSuffix>PA-3305</VersionSuffix>
```

Comment:
Assuming you branched off of develop it's a bit concerning that the develop branch has a pre-release tag in it. According to our SW those should always be removed prior to merging to develop.